lib/bot.js

```diff
@@ -33,6 +34,25 @@ class Bot {
     this.ircNickColor = options.ircNickColor !== false; // default to true
     this.channels = _.values(options.channelMapping);
 
+    this.formatCommandPrelude = options.formatCommandPrelude || 'Command sent from Discord by {$nickname}:';
```

Comment:
Maybe create a `format` object and have these as properties? I.e.
```js
"format": {
  "commandPrelude": "...",
  "ircText": "...",
  "urlAttachment": "..."
}
```

or something like that? Then just add a `this.format = options.format || {};` above?